docs_app/content/deprecations/to-promise.md

```diff
@@ -0,0 +1,99 @@
+# Conversion to Promises
+
+The similarity between Observables and Promises is that both [collections](/guide/observable) may produce values over
+time, but the difference is that Observables may produce none or more than one value, while Promises produce only one
+value when resolved successfully.
+
+# Issues
+
+For this reason, in RxJS 7, the return type of the Observable's [`toPromise()`](/api/index/class/Observable#toPromise)
+method has been fixed to better reflect the fact that Observables can yield zero values. This may be a **breaking
+change** to some projects as the return type was changed from `Promise<T>` to `Promise<T | undefined>`.
+
+Also, `toPromise()` method name was never indicating what emitted value a Promise will resolve with because Observables
+can produce multiple values over time. When converting to a Promise, you might want to choose which value to pick -
+either the first value that has arrived or the last one. To fix all these issues, we decided to deprecate `toPromise()`,
+and to introduce the two new helper functions for conversion to Promises.
+
+# Use one of the two new functions
+
+As a replacement to the deprecated `toPromise()` method, you should use one of the two built in static conversion
+functions {@link firstValueFrom} or {@link lastValueFrom}.
+
+## `lastValueFrom`
+
+The `lastValueFrom` is almost exactly the same as `toPromise()` meaning that it will resolve with the last value that has
+arrived when the Observable completes, but with the difference in behavior when Observable completes without emitting a
+single value. When Observable completes without emitting, `toPromise()` will successfully resolve with `undefined` (thus
+the return type change), while the `lastValueFrom` will reject with the {@link EmptyError}. Thus, the return type of the
```

Comment:
IDK whether or not the TSDoc-style `{@link whatever}` tags can be used. I would be surprised if the tooling were to turn them into links. Are there examples of this elsewhere? Does it work? In the multicasting docs that I wrote, I just used Markdown links:

```
And [shareReplay](/api/operators/shareReplay) - which is a thin wrapper around the now highly-configurable [share](/api/operators/share) operator.
```

Reply:
Actually, they can:

![image](https://user-images.githubusercontent.com/28087049/116562492-7ce04280-a903-11eb-9f19-e0279c3cf8ad.png)

Another example:
https://github.com/ReactiveX/rxjs/commit/539b74281eafde149e473f0aeed8a1f75c8ecf9f#diff-859daf5e87a340e04295a22f9ba74ef5a6508fbd7fe99e8fb0a1ecad87b15b3fR156

This is not yet visible at next.rxjs.dev, so here's the excerpt from the local run:

![image](https://user-images.githubusercontent.com/28087049/116563306-32ab9100-a904-11eb-8944-a61aba021de4.png)

Reply:
TIL. This would have saved me some typing. 😅